site/content/api/Event-API.md

```diff
@@ -76,4 +81,56 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+### Lifecycle Notifications
+
+To enable *Business Applications* to react to the online status of devices belonging to a specific tenant, a special type of event - named *lifecycle notification* - is predefined.
```

Comment:
I would rather not use the term *online* in this context, simply because there is no definition of what it means to be *online*. AFAIC we should refrain to talking about a device's connection status, i.e. we can detect when a device connects to one of the protocol adapters and we *may* be able to detect when it disconnects. I do not want to open up a wide field of speculation about what should and what shouldn't be able on a device that is supposedly *online*.